### PR TITLE
RECRUITMENT_BANNER_* env vars for site-wide banner

### PR DESCRIPTION
**Pull request by groovecoder**
This works, but it also pushes the `header-top` element down and overlaps with the `<main>` element? @lesleyjanenorton - do you know a quick fix?

image

### CODE REVIEW

File: privaterelay/settings.py [hunk 1]
```diff
@@ -99,14 +99,12 @@
 AWS_SNS_TOPIC = config('AWS_SNS_TOPIC', None)
 AWS_SNS_KEY_CACHE = config('AWS_SNS_KEY_CACHE', 'default')
 AWS_SES_CONFIGSET = config('AWS_SES_CONFIGSET', None)
-SOCKETLABS_SERVER_ID = config('SOCKETLABS_SERVER_ID', 0, cast=int)
-SOCKETLABS_API_KEY = config('SOCKETLABS_API_KEY', None)
-SOCKETLABS_SECRET_KEY = config('SOCKETLABS_SECRET_KEY', None)
-SOCKETLABS_VALIDATION_KEY = config('SOCKETLABS_VALIDATION_KEY', None)
```
Comment (pdehaan) [inline privaterelay/settings.py:105]: Did you mean to nuke the socketlabs configs? Seems unrelated to a recruitment banner.

Comment (groovecoder) [reply privaterelay/settings.py:105]: Sorry, yes I should have commented here. The `SOCKETLABS` stuff should be removed ASAP. I could split into 2 commits maybe?

Comment (pdehaan) [reply privaterelay/settings.py:105]: Or, do a separate PR for socket labs and I'll blindly R+ and merge immediately, and then we won't be blocked on banner UI stuff, would also work.

**Comment by groovecoder** [comment]
Added a `recruitmentLogic` function which stops any recruitment link from showing up for 30 days after someone clicks a recruitment link.

File: privaterelay/settings.py (hunk 1 above)
Comment (groovecoder) [reply privaterelay/settings.py:105]: I'll make it a separate PR/change.

Comment (groovecoder) [reply privaterelay/settings.py]: Updated.

**Comment by say-yawn** [comment]
This is blocking the survey we want to deploy so getting it done this week will be helpful. @lesleyjanenorton volunteered to design if we cannot get DPX specs for this.

**Comment by groovecoder** [comment]
@lesleyjanenorton I'm seeing this now? Is my CSS cache just stale?
image

File: privaterelay/templates/includes/header.html [hunk 2]
```diff
@@ -1,5 +1,16 @@
+{% load i18n %}
 {% load static %}
+
+{% get_current_language as LANGUAGE_CODE %}
+
 <header class="flx flx-col spc-btwn">
+
+  <div class="recruitment-banner">
+    {% if settings.RECRUITMENT_BANNER_LINK and LANGUAGE_CODE|slice:"0:2" == "en" %}
+      <a id="recruitment-banner" class="text-link" href="{{ settings.RECRUITMENT_BANNER_LINK }}" target="_blank" rel="noopener noreferrer" data-ga="send-ga-funnel-pings" data-event-category="Recruitment" data-event-label="{{ settings.RECRUITMENT_BANNER_TEXT }}">{{ settings.RECRUITMENT_BANNER_TEXT }}</a>
```
Comment (pdehaan) [inline privaterelay/templates/includes/header.html:10]: Do we need to escape the `data-event-label` attribute, like we do on https://github.com/mozilla/blurts-server/pull/1957/files#diff-d803a6e07903be9fc08d03c1e5bdc59f6513e8f385d8b482df0f0991bcf91dd3R44?

**Comment by lesleyjanenorton** [comment]
@groovecoder updated! Not sure what happened.

File: privaterelay/templates/includes/header.html (hunk 2 above)
Comment (groovecoder) [reply privaterelay/templates/includes/header.html:10]: No need - Django templates [automatically escape HTML](https://docs.djangoproject.com/en/3.1/ref/templates/language/#automatic-html-escaping).

**Comment by sbadau** [comment]
Verified on Mac OS X 10.15, Windows 10 x64, and Ubuntu 20.04 x64 that:
- the banner containing the text "Are 5 email aliases in Relay enough? Take this survey and let us know." is displayed when accessing the Firefox Relay website
- the banner disappears if clicked upon
- the banner reappears after 30 days.

However, while verifying I stumbled upon 2 new issues:
1. After clicking on the banner - the banner is shortly displayed on each refresh of the Firefox Relay website. Please see the screencast for more details - https://imgur.com/a/gcb3se5
2. After clicking on the banner - if signing out of the Firefox relay account, the banner is displayed (you don't have to wait for 30 days). Please see the screencast for more details - https://imgur.com/a/itEV7EP

@groovecoder @lesleyjanenorton Please let me know if I should file new reports for the 2 mentioned issues. Thanks!
